src/spicy/spicyz/glue-compiler.cc

```diff
@@ -989,7 +989,8 @@ bool GlueCompiler::compile() {
     if ( ! _doc_id.empty() ) {
         auto mtime = hilti::expression::Ctor(hilti::ctor::Time(hilti::rt::time::current_time()));
         preinit_body.addCall("zeek_rt::register_spicy_module_begin",
-                             {hilti::builder::string(_doc_id), hilti::builder::string(_doc_description), mtime});
+                             {hilti::builder::string_mut(_doc_id), hilti::builder::string_mut(_doc_description),
```

Comment:
Bit unsure why the spicy changes ended up here, but can probably straightened out during a final rebase or merge.

Reply:
Just pushed fixups for the `Trigger` stuff, with luck it's now sound.

The spicy changes may have been a cut&paste glitch on copying a commit hash. If not, then I Have No Idea